code/modules/reagents/chemistry/machinery/chem_master.dm

```diff
@@ -98,15 +98,16 @@
 		return TRUE
 
 	if((istype(I, /obj/item/reagent_containers/glass) || istype(I, /obj/item/reagent_containers/food/drinks/drinkingglass)) && user.a_intent != INTENT_HARM)
-		if(beaker)
-			to_chat(user, "<span class='warning'>A beaker is already loaded into the machine.</span>")
-			return
 		if(!user.drop_item())
 			to_chat(user, "<span class='warning'>[I] is stuck to you!</span>")
 			return
-		beaker = I
 		I.forceMove(src)
-		to_chat(user, "<span class='notice'>You add the beaker to the machine!</span>")
+		if(beaker)
+			user.put_in_hands(beaker)
+			to_chat(user, "<span class='notice'>You swap [I] with [beaker].</span>")
+		else
+			to_chat(user, "<span class='notice'>You set [I] on the machine.</span>")
```

Comment:
Can you change this back to its old to_chat, or add a small bit about it to your PR desc? To me, I dont think theres a spot "on" the machine to put it, I always thought you put it in the machine

Reply:
would [this](https://github.com/ParadiseSS13/Paradise/pull/22915/commits/dccf107e5b0d8784658f6e4e1f0bcdc68f4fa330) do?

Reply:
Yeah sure, just needs a slight altering with the `!` in the notice span, I can put up a suggestion